README.md

```diff
@@ -200,11 +266,143 @@ df <- read.df(table_path, "deltaSharing")
 
 You can try this by running our [examples](examples/README.md) with the open, example Delta Sharing Server.
 
+### CDF 
+Starting from release 0.5.0, querying [Change Data Feed](https://docs.databricks.com/delta/delta-change-data-feed.html) is supported with delta sharing.
+Once the provider turn on CDF on the original delta table and share it through delta sharing. Once CDF is shared, the recipient can query
+CDF of a delta sharing table similar to CDF of a delta table.
+```scala
+val tablePath = "<profile-file-path>#<share-name>.<schema-name>.<table-name>"
+val df = spark.read.format("deltaSharing")
+  .option("readChangeFeed", "true")
+  .option("startingVersion", "3")
+  .load(tablePath)
+```
+
+### Streaming
+Starting from release 0.6.0, delta sharing table can be used as the data source for [spark structured streaming](https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html).
+Once the provider shares a table with history, the recipient can perform a streaming query on the table.
+```scala
+val tablePath = "<profile-file-path>#<share-name>.<schema-name>.<table-name>"
+val df = spark.readStream.format("deltaSharing")
+  .option("startingVersion", "1")
+  .option("ignoreChanges", "true")
+  .load(tablePath)
+```
+
 ### Table paths
 
 - A profile file path can be any URL supported by Hadoop FileSystem (such as `s3a://my_bucket/my/profile/file`).
 - A table path is the profile file path following with `#` and the fully qualified name of a table (`<share-name>.<schema-name>.<table-name>`).
 
+# Other Connectors
+<table>
+<tr>
+<th>Connector</th>
+<th>GitHub Link</th>
+<th>Status</th>
+<th>Supported Features</th>
+</tr>
+<tr>
+<td>Power BI</td>
+<td>Databricks owned</td>
+<td>Released</td>
+<td>QueryTableVersion<br>QeuryTableMetadata<br>QueryTableLatestSnapshot</td>
+</tr>
+<tr>
+<td>Node.js</td>
+<td>
+
```

Comment:
Weird way to segment the sections? Should it be split by project instead of like this?

Reply:
Perhaps we could include a nice graphic in this section similar to this [one](https://media.licdn.com/dms/image/C4E22AQE7ZSARtLnavg/feedshare-shrink_2048_1536/0/1671113944328?e=1675900800&v=beta&t=2tgTHaxlJW-ziiK7RwxEgzABP8NpuhvN2b0HbKU0zH8)?

Reply:
I'll check how to inject a pic in the text.

Reply:
added the pic.